s4e-backend/src/main/java/pl/cyfronet/s4e/bean/Invitation.java

```diff
@@ -0,0 +1,41 @@
+package pl.cyfronet.s4e.bean;
+
+import lombok.*;
+import pl.cyfronet.s4e.bean.audit.CreationAndModificationAudited;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "invitation", uniqueConstraints = @UniqueConstraint(columnNames = {"email", "institution_id"}))
```

Comment:
The `@Table` annotation here isn't necessary, as:
* the class name matches the table name
* constraint will be applied in a db migration directly (this one will be ignored)